app/experimenter/projects/models.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+
+
+class Project(models.Model):
+    name = models.CharField(max_length=255, unique=True, blank=False, null=False)
+    slug = models.SlugField(max_length=255, unique=True, blank=False, null=False)
```

Comment:
I think `blank=False, null=False` are the defaults so it's redundant?  If I'm wrong just leave it.